### ISSUE TITLE
serialVersionUID should be ignored in CONSTANT_UPPERCASE

### ISSUE THREAD

**Issue by illarionov** [saveourtool/diktat#1847]
## Describe the bug
This code:
```kotlin
class TestSerializableClass() : Serializable
    companion object {
        private const val serialVersionUID: Long = -1
    }
}
```
triggers the error: 
```
[CONSTANT_UPPERCASE] <val> properties from companion object or on file level mostly in all cases are constants - please use upper snake case for them: serialVersionUID (diktat-ruleset:identifier-naming)
```

## Expected behavior
serialVersionUID should be ignored

## Environment information
* diktat version: 1.2.5
* build tool (maven/gradle): gradle
* how is diktat run (CLI, plugin, etc.): Gradle plugin
* kotlin version: 1.9.21
* operating system:
* link to a project (if your project is public):

**Comment by kgevorkyan** [comment]
Hello @illarionov!

Thanks for reporting, but I'm not sure. whether there is a error

However, you can use `@Suppress` annotation, to ignore this rule, if you want to use this variable directly with this name

```kotlin
    companion object {
        @Suppress("CONSTANT_UPPERCASE")
        private const val serialVersionUID: Long = -1
    }
```

**Comment by illarionov** [comment]
This is a feature request, actually.  
`serialVersionUID" this is the field name that cannot be changed when implementing the Serializable interface. 
Adding Suppress annotation to every field impair readability.

**Comment by akuleshov7** [comment]
Good point actually. This can be harcoded easily in naming rule. 
This is a good corner case, may be can be reflected in our guide https://github.com/saveourtool/diktat/blob/master/info/guide/diktat-coding-convention.md

**Comment by akuleshov7** [comment]
> Hello @illarionov!
> 
> Thanks for reporting, but I'm not sure. whether there is a error
> 
> However, you can use `@Suppress` annotation, to ignore this rule, if you want to use this variable directly with this name
> 
> ```kotlin
>     companion object {
>         @Suppress("CONSTANT_UPPERCASE")
>         private const val serialVersionUID: Long = -1
>     }
> ```

the same can be done by `@get:JvmName("serialVersionUID")`, but yes, it looks ugly and not user friendly
